lib/api/programsLogic.ts

```diff
@@ -0,0 +1,22 @@
+import prisma from '../../prisma/prisma';
+import { programsWriteSchema } from '../types/dtos/createProgramRequest.dto';
+import { BadRequestError } from '../utils/errors/badRequestError';
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+export const getNewsHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+  res.status(200).json({ news: await prisma.program.findMany() });
+};
+
+export const postProgramsHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+  const validatedProgram = validateCreateProgramRequest(req.body);
```

Comment:
Should we have all of our functions use "Programs" with an "s" for consistency? Or should we update this function name to be "postProgramHandler"?

Reply:
I like having the "s" because it's labeled "Programs" on the site like that. thoughts?